### PR TITLE
added dependency, updated form

### PR DESCRIPTION
**Pull request by gandesc**

### CODE REVIEW

File: src/bulma/pages/products/components/BaseForm.vue [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <template>
     <enso-form class="box form-box has-background-light raises-on-hover"
         @ready="form = $event.form">
+        <template v-slot:category_id="{ field }">
+            <label class="label">
+                {{ i18n('Category') }}
+            </label>
+            <categories v-model="field.value"
+                ref="categories"/>
+        </template>
```
Comment (aocneanu) [inline src/bulma/pages/products/components/BaseForm.vue:15]: there's more here to connect .. look in the FormField as example

Comment (gandesc) [reply src/bulma/pages/products/components/BaseForm.vue:15]: updated

File: src/bulma/pages/products/components/BaseForm.vue [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
         @ready="form = $event.form">
-        <template v-slot:category_id="{ field }">
+        <template v-slot:category_id="{ field, errors }">
             <label class="label">
                 {{ i18n('Category') }}
+                <span class="icon is-small has-text-info"
+                    v-tooltip="i18n(field.meta.tooltip)"
+                    v-if="field.meta.tooltip">
+                    <fa icon="info-circle"
+                        size="xs"/>
```
Comment (aocneanu) [inline src/bulma/pages/products/components/BaseForm.vue]: I somebody need to add a tooltip (small chances) here would have to customize the Form object. In that case he can also customize the view. Let's remove it for now

File: src/bulma/pages/products/components/BaseForm.vue [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
+                </span>
             </label>
             <categories v-model="field.value"
```
Comment (aocneanu) [inline src/bulma/pages/products/components/BaseForm.vue:8]: misses `@input="errors.clear(field.name); $emit('changed')"`

**Pull request merged**